.github/workflows/docker-image-release.yml

```
@@ -0,0 +1,49 @@
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  push_to_registries:
+    name: Push Docker image to multiple registries
```

```suggestion
    name: Push Docker image to registry
```

Appears to be just GHCR?

---

originally had this as pushing to docker hub too but I was unsure what you wanted to do here (or if you had a dockerhub account on your end), so I guess my question is what do you prefer?

---

I think I prefer just GHCR for now :)